src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingActualsController.java

```diff
@@ -27,22 +27,23 @@ public class HearingActualsController {
     private final AccessControlService accessControlService;
 
     public HearingActualsController(HearingActualsService hearingActualsService,
-                                    AccessControlService accessControlService) {
+            AccessControlService accessControlService) {
         this.hearingActualsService = hearingActualsService;
         this.accessControlService = accessControlService;
     }
 
     @GetMapping(path = "/hearingActuals/{id}", produces = APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    @ApiResponses(value = {
-        @ApiResponse(code = 204, message = "Hearing id is valid"),
-        @ApiResponse(code = 404, message = ValidationError.HEARING_ID_NOT_FOUND),
-        @ApiResponse(code = 400, message = ValidationError.INVALID_HEARING_ID_DETAILS)
+    @ApiResponses({
+        @ApiResponse(responseCode = "204", description = "Hearing id is valid"),
+        @ApiResponse(responseCode = "404", description = ValidationError.HEARING_ID_NOT_FOUND),
+        @ApiResponse(responseCode = "400", description = ValidationError.INVALID_HEARING_ID_DETAILS)
     })
```

Comment:
I'd suggest to remove wrapper, and leave just this:
```
@ApiResponse(responseCode = "204", description = "Hearing id is valid")
@ApiResponse(responseCode = "404", description = ValidationError.HEARING_ID_NOT_FOUND)
@ApiResponse(responseCode = "400", description = ValidationError.INVALID_HEARING_ID_DETAILS)
```
I believe it is picked up by sonar as a code smell.

Reply:
its best practice, and passes test so i think ill leave it in, also keeps diff smaller

Reply:
I don't see how is it a best practice if it is marked as code smell.. This was required with springfox but is undesirable (though allowed) with springdoc (https://github.com/joansmith/sonar-java/blob/master/java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1710.html). I am fine either way.